pkg/replica/server.go

```diff
@@ -113,7 +115,19 @@ func (s *Server) Status() (State, Info) {
 		info, err := ReadInfo(s.dir)
 		if os.IsNotExist(err) {
 			return Initial, Info{}
-		} else if err != nil {
+		}
+		if errors.Is(err, io.EOF) {
+			volumeMetaFilePath := (&Replica{dir: s.dir}).diskPath(volumeMetaData)
+			fileInfo, statErr := os.Stat(volumeMetaFilePath)
+			if statErr != nil {
+				logrus.Errorf("Failed to read info in replica directory %s: %v", s.dir, err)
+				return Error, Info{}
+			} else if fileInfo.Size() == 0 {
+				os.Remove(volumeMetaFilePath)
+				return Initial, Info{}
+			}
+		}
```

Comment:
Should not do any `change` operation in `get`-style function/method.

Reply:
Removed `os.Remove(volumeMetaFilePath)`. Thanks.